openfisca_tracker/piwik.py

```diff
@@ -34,11 +29,9 @@ def stop_timer(self):
 
     def send(self):
         with self.lock:
-            post(
+            requests.post(
```

Comment:
Using [grequests](https://pypi.org/project/grequests/), the syntax looks like this:
```
            rs = (
                grequests.post(self.url, json={"requests": request, "token_auth": self.token_auth})
                for request in self.requests
                )
            grequests.map(rs)
```